src/cdk-experimental/combobox/combobox.ts

```diff
@@ -6,15 +6,192 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Directive} from '@angular/core';
+export type OpenAction = 'focus' | 'click' | 'downKey' | 'toggle';
+
+export type OpenActionInput = OpenAction | OpenAction[] | string | null | undefined;
+
+import {
+  AfterContentInit,
+  Directive,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  Optional,
+  Output, ViewContainerRef
+} from '@angular/core';
+import {CdkComboboxPanel} from './combobox-panel';
+import {TemplatePortal} from '@angular/cdk/portal';
+import {
+  ConnectedPosition,
+  FlexibleConnectedPositionStrategy,
+  Overlay,
+  OverlayConfig,
+  OverlayRef
+} from '@angular/cdk/overlay';
+import {Directionality} from '@angular/cdk/bidi';
+import {BooleanInput, coerceBooleanProperty, coerceArray} from '@angular/cdk/coercion';
+
 
 @Directive({
   selector: '[cdkCombobox]',
   exportAs: 'cdkCombobox',
   host: {
-    'role': 'combobox'
+    'role': 'combobox',
+    '(click)': 'toggle()',
+    '[attr.aria-disabled]': 'disabled'
   }
 })
-export class CdkCombobox<T = unknown> {
+export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
+  @Input('triggerFor')
+  get comboboxPanel(): CdkComboboxPanel<T> | undefined {
+    return this._comboboxPanel;
+  }
+  set comboboxPanel(panel: CdkComboboxPanel<T> | undefined) {
+    this._comboboxPanel = panel;
+  }
+  private _comboboxPanel: CdkComboboxPanel<T> | undefined;
+
+  @Input()
+  get value(): T {
+    return this._value;
+  }
+  set value(val: T) {
+    this._value = val;
+  }
+  private _value: T;
+
+  @Input()
+  get disabled(): boolean {
+    return this._disabled;
+  }
+  set disabled(value: boolean) {
+    this._disabled = coerceBooleanProperty(value);
+  }
+  private _disabled: boolean = false;
```

Comment:
optional: you can condense simple getter/setters like this
```suggestion
  get disabled(): boolean { return this._disabled; }
  set disabled(value: boolean) { this._disabled = coerceBooleanProperty(value); }
  private _disabled: boolean = false;
```